### ISSUE TITLE
Get Broken Pipe SocketException after exit brace.

### ISSUE THREAD

**Issue by rocky-peng** [btraceio/btrace#250]
Every time after I quit brace application by 'Ctrl+C' and chose '1', the application which was btraced will  get the exception,and it will continue occurs.

The full stack :

```
java.net.SocketException: Broken pipe
        at java.net.SocketOutputStream.socketWrite0(Native Method)
        at java.net.SocketOutputStream.socketWrite(SocketOutputStream.java:113)
        at java.net.SocketOutputStream.write(SocketOutputStream.java:159)
        at java.io.ObjectOutputStream$BlockDataOutputStream.writeBlockHeader(ObjectOutputStream.java:1889)
        at java.io.ObjectOutputStream$BlockDataOutputStream.drain(ObjectOutputStream.java:1874)
        at java.io.ObjectOutputStream$BlockDataOutputStream.setBlockDataMode(ObjectOutputStream.java:1785)
        at java.io.ObjectOutputStream.reset(ObjectOutputStream.java:499)
        at com.sun.btrace.agent.RemoteClient.onCommand(RemoteClient.java:137)
        at com.sun.btrace.BTraceRuntime$ConsumerWrapper.accept(BTraceRuntime.java:169)
        at com.sun.btrace.BTraceRuntime$ConsumerWrapper.accept(BTraceRuntime.java:157)
        at com.sun.btrace.org.jctools.queues.MpscChunkedArrayQueue.drain(MpscChunkedArrayQueue.java:624)
        at com.sun.btrace.BTraceRuntime$4.run(BTraceRuntime.java:443)
        at java.lang.Thread.run(Thread.java:745)                                           
```

**Comment by jbachorik** [comment]
Which BTrace version?
There was a bug fixed with a similar root cause for 1.3.8.2 - #250

**Comment by rocky-peng** [comment]
Thank you very much. The version of btrace I used is 1.3.8.1 and after I switch to 1.3.8.3,this problem has gone.
